.gitlab-ci.yml

```diff
@@ -377,7 +377,7 @@ iwyu:
 
 # Code analysis, coverage and reporting tool
 # For short living branches or PRs, try to detect an open PR
-sonarqube_cov:
+sonarqube_cov_:
```

Comment:
Is there a reason why this is the only name with an underscore as a postfix?

Reply:
The name needs to be different otherwise only one definition of these exist. See for example [this pipeline where there is no sonarqube_cov job](https://gitlab.com/ginkgo-project/ginkgo-public-ci/pipelines). The job which is kept seems to be the one for the long living branches. I wanted to create a new name while keeping something similar, because the jobs are different only in whether or not they apply to long or short living branches.

Reply:
Thanks, that makes sense.
I just also saw that you wrote that in the description, sorry for not reading that first.